Ugizwenayo-Divine/FoodStuffPriceCompare-App
# chore(settingUpTest): unit testing

#### What does this PR do?
- setting up unit testing
#### Description of Task to be completed?
- installed required packages(mocha,chai,chai-http)
- created index.test.js file
- wrote the testing code
- wrote the run test script
#### How should this be manually tested?
- After cloning the project repo, run npm i inside the cloned repo folder
- change in cloned repo
- git fetch
- git checkout ch-setup-unit-test
- npm i
- run the npm run test you must see the test passing
#### Any background context you want to provide?
- you must have node installed locally
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
